### PR TITLE
DTSPO-900 Deploy mailrelay to dev

### PR DESCRIPTION
**Pull request by luigibk**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/release/admin/mail-relay/mailrelay.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: mailrelay
+  namespace: admin
```
Comment (timja) [inline k8s/release/admin/mail-relay/mailrelay.yaml:6]: shouldn't this be in it's own namespace?

**Review by timja — APPROVED** [review]

File: k8s/release/admin/mail-relay/mailrelay.yaml (hunk 1 above)
Comment (luigibk) [reply k8s/release/admin/mail-relay/mailrelay.yaml:6]: A mail relay si quite an "adminy" thing ... but yeah I've been asking myself the same question.

File: k8s/common/mailrelay.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: v1
+kind: Namespace
```
Comment (timja) [inline k8s/common/mailrelay.yaml:3]: can you create this as part of the kustomize base, `common` shouldn't be used for anything new

File: k8s/release/mailrelay/mailrelay/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- mailrelay.yaml
```
Comment (timja) [inline k8s/release/mailrelay/mailrelay/kustomization.yaml:4]:
```suggestion
- namespace.yaml
- mailrelay.yaml
```

File: k8s/common/mailrelay.yaml (hunk 2 above)
Comment (luigibk) [reply k8s/common/mailrelay.yaml:3]: Yes, had not seen that. Thanks

**Pull request merged**